WeakAuras/Prototypes.lua

```diff
@@ -11244,9 +11288,160 @@ Private.event_prototypes = {
           end
         ]=]
       end
-      return ret:format(trigger.currencyId or 1)
+
+      ret = ret .. [=[
+        -- Gather currency data for account characters
+        if currencyInfo and currencyInfo.isAccountTransferable then
```

Comment:
this whole code probably should check for "clone" somewhere

Reply:
Even if clones are disabled, the trigger will still calculate values like `accountQuantity` and `realAccountQuantity`. 
I could add a "Fetch account data" option to the trigger aside from "Clone per character", but I don't think that is necessary.